conf/config-default.yaml

```diff
@@ -14,122 +14,121 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-# PLEASE DO NOT UPDATE THIS FILE!
-# If you want to set the specified configuration value, you can set the new
-# value in the conf/config.yaml file.
+# CAUTION: DO NOT MODIFY DEFAULT CONFIGURATIONS IN THIS FILE.
+# Keep the custom configurations in conf/config.yaml.
 #
 
 apisix:
-  # node_listen: 9080               # APISIX listening port
-  node_listen:                      # This style support multiple ports
+  # node_listen: 9080          # APISIX listening port.
+  node_listen:                 # APISIX listening ports.
     - 9080
-  #   - port: 9081
-  #     enable_http2: true          # If not set, the default value is `false`.
-  #   - ip: 127.0.0.2               # Specific IP, If not set, the default value is `0.0.0.0`.
-  #     port: 9082
-  #     enable_http2: true
+    # - port: 9081
+    #   enable_http2: true     # If not set, default to `false`.
+    # - ip: 127.0.0.2          # If not set, default to `0.0.0.0`/
+    #   port: 9082
+    #   enable_http2: true
```

Comment:
Could you provide some examples where # at the far left is considered a better practice?

Reply:
I don't know if it is a better practice but autoformatters I use do this. I thought that was how it is done in general.

Reply:
Won't fixed at this time because the position of `#` depends on the left most line of code in selection. As there are a lot of indented configurations that need to be commented out here, the `#` signs are indented as well. It would take a lot of manual effort to have them all on the far left (which the original file did not do either).